src/components/Settings/ResetAccountButton.tsx

```diff
@@ -0,0 +1,89 @@
+import { Dialog, Transition } from "@headlessui/react";
+import { IoIosWarning, IoIosArrowBack } from "react-icons/io";
+import { Fragment, useState } from "react";
+import { BiReset } from "react-icons/bi";
+import WarningButton from "./WarningButton";
+import Importer from "../../helpers/Importer";
+
+function ResetAccountButton() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  const handleReset = () => {
+    Importer.init();
+    setIsOpen(true);
+  };
+
+  const handleClose = () => {
+    window.location.pathname = "/settings";
+  };
+
+  return (
+    <>
+      <WarningButton
+        name="Reset"
+        icon={<BiReset />}
```

Comment:
Didn't we use Font Awesome elsewhere? Why the library change?

Reply:
This is react icons under font awesome instead of `react-icons/fa` it's `react-icons/bi`

Reply:
Same dependency that is

Reply:
Not a fan of inconsistent design languages but we're just gonna roll with it due to time constraints. Fix as stretch goal.